static/js/around-map.js

```diff
@@ -6,39 +6,40 @@ const map = new mapboxgl.Map({
   zoom: 2
 })
 
-// users interesting points
-const pois = new Worker('/static/js/webworker-around.js');
-
-pois.onmessage = informations => {
-  const [name, wifi, power, comment, lng, lat] = informations.data
+map.addControl(new mapboxgl.NavigationControl())
+map.addControl(new mapboxgl.GeolocateControl())
 
-  const el = document.createElement('div')
-  el.classList.add('marker')
+// users interesting points
+const worker = new Worker('/static/js/webworker-around.js')
 
-  let content = `<h2>Hello ${name}</h2>
-                 <ul>
-                    <li>Wifi quality: ${wifi}</li>
-                    <li>Power available: ${power}</li>
-                    <li>Comments: ${comment}</li>
-                  </ul>`
+worker.onmessage = response => {
+  response.data.forEach(poi => {
+    const el = document.createElement('div')
+    el.classList.add('marker')
 
-  const popup = new mapboxgl.Popup({offset: [10, -20]})
-    .setHTML(content)
+    const popup = new mapboxgl.Popup({offset: [10, -20]})
+      .setHTML(`<h2>Hello ${poi.name}</h2>
+                   <ul>
+                      <li>Wifi quality: ${poi.wifiQuality}</li>
+                      <li>Power available: ${poi.powerAvailable}</li>
+                    </ul>`
+            )
 
-  new mapboxgl.Marker(el, {offset:[0, -30]})
-    .setLngLat([lng, lat])
-    .setPopup(popup)
-    .addTo(map)
-};
+    new mapboxgl.Marker(el, {offset:[4, -6})
```

Comment:
`new mapboxgl.Marker(el, {offset: [4, -6]})`
A space after the comma and the closing bracket is missing